src/renderer/common/js/heightChangeListener.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * A component which detects height modifications of the parent DOM element
+ *
+ * This component should always have an iframe as a container. It fires an event whenever
+ * the height of the parent DOM element changes.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+    /**
+     * Responsible for detecting height changes in the DOM element in which
+     * the component's container is nested.
+     */
+    fluid.defaults("gpii.psp.heightChangeListener", {
+        gradeNames: ["fluid.viewComponent"],
+        listeners: {
+            "onCreate.initResizeListener": {
+                "this": "@expand:$({that}.container.0.contentWindow)",
```

Comment:
Binding to the $ symbol as a global is generally dangerous in a browser context since this is intended to be a lightweight symbol which can be rebound within a closure to whichever library is most convenient for the local implementation. I know that you have 100% control over the loading context within the markup but I suggest that just for general clarity this is written as "jQuery"